### PR TITLE
Total anomaly change

### PR DESCRIPTION
**Pull request by henrikny**
This PR will apply the following changes:

- remove unnecessary function for the calculation for the feature thresholds (easier to read)
- new calculation of total anomaly (scaled and unscaled analogue):
from 

`data["total-anomaly-scaled"] = np.linalg.norm(data["tag-anomaly-scaled"], axis=1)`
to
`data["total-anomaly-scaled"] = np.square(data["tag-anomaly-scaled"]).mean(axis=1)`

- the total model threshold validation is simplified

### CODE REVIEW

**Comment by epa095** [comment]
The build-doc tests fail because of https://github.com/equinor/gordo/issues/927 . When that is fixed and merged you need to rebase on master, and then that test should pass. 

Then there is the `machine` test, which you need to change according to your changes. The easiest is probably for you to run the tests locally. `pip install -r requirements/test_requirements.txt` and then `python setup.py testmachine` to run the test-suite.

File: tests/gordo/machine/model/anomaly/test_anomaly_detectors.py [hunk 1]
```diff
@@ -209,27 +209,6 @@ def test_diff_detector_cross_validate(return_estimator: bool):
     assert cv_results_da.keys() == cv_results_sk.keys()
 
 
-# simulate LSTM outptu shorter than input
```
Comment (flikka) [inline tests/gordo/machine/model/anomaly/test_anomaly_detectors.py]: Why is this removed?

**Review by epa095 — APPROVED** [review]
LGTM :+1:
